src/v2/Apps/Consign/Components/InDemandNow/DemandRank.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import {
+  Text as BaseText,
+  Box,
+  Flex,
+  Spacer,
+  TriangleDownIcon,
+} from "@artsy/palette"
+import styled from "styled-components"
+
+export const DemandRank: React.FC<{ demandRank: number }> = ({
+  demandRank,
+}) => {
+  let width = demandRank * 100
```

Comment:
(non-blocking suggestion) You could eliminate the `if` statement below using `Math.min`:
```suggestion
  let width = Math.min(demandRank * 100, 100)
```